### PR TITLE
Feature/node8.2.1

### PR DESCRIPTION
**Pull request by avnersorek**
relates to https://github.com/dorbel-tech/dorbel-app/issues/1387

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,18 @@
-# Docker hub repository name: dorbel/alpine-node
+# Dorbel base image for node Docker containers
 
-The repository is located here: https://hub.docker.com/r/dorbel/alpine-node/
+## Docker hub repository
+* Name: dorbel/alpine-node
+* The repository is located here: https://hub.docker.com/r/dorbel/alpine-node/
+
+## Docker hub authentication
+* You will a Docker Hub (https://hub.docker.com/) user and password, and also be granted permissions to the dorbel/alpine-node image.
```
Comment (virtser) [inline README.md]: You will **need** ...

**Review by virtser — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -1,7 +1,13 @@
-FROM mhart/alpine-node:6.10.3
+FROM mhart/alpine-node:8.2.1
 
-RUN apk add --no-cache git bash wget curl nano
-RUN npm install --global yarn@v0.24.5 --no-progress
+RUN apk add --no-cache git bash wget curl nano python make g++
```
Comment (virtser) [inline Dockerfile:3]: Do we really need `python make g++` there?
The size of image more than doubled.

dorbel_alpine-node_-_docker_hub

Comment (avnersorek) [reply Dockerfile:3]: yes :cry: 
It doesn't work without it

Comment (virtser) [reply Dockerfile:3]: what doesnt work without it?

Comment (avnersorek) [reply Dockerfile:3]: The build of some of our dependencies (the ``yarn install`` step fails)

**Pull request merged**